schemas/meta/schema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "definitions": {
+        "field": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "description": "The type of field",
+                    "type": "string",
+                    "enum": [
+                        "date",
+                        "text",
+                        "html"
+                    ]
+                }
+            },
+            "required": [
+                "type"
+            ],
+            "additionalProperties": false
+        }
+    },
+    "type": "object",
+    "properties": {
+        "title": {
+            "description": "The subject's title.",
+            "type": "string"
+        },
+        "slug": {
```

Comment:
I took a liberty of naming this `slug` instead of `postType` (as is defined in the issue) because it felt appropriate.

Reply:
`slug` is indeed the better choice 👍